src/types/apiTypes.ts

```diff
@@ -344,6 +344,11 @@ const zComfyComboOutput = z.array(z.any())
 const zComfyOutputTypesSpec = z.array(
   z.union([zComfyNodeDataType, zComfyComboOutput])
 )
+const zDescriptionSpec = z.union([
```

Comment:
Can you provide links to `DESCRIPTION` field used in custom nodes in this format?

Reply:
A new format is being established here, so no node currently makes use of it. Node packs with a la carte documentation code currently need to play hot potato with swapping out the node def during load so that full descriptions aren't used for node previews or title hover tooltips. There should be a way for a node to describe documentation in place without requiring JS of any form, let alone one that requires abuse of the standards in place.

Test cases have been added for this format.